versions.props

```diff
@@ -68,6 +68,8 @@
       -->
       8.0.*
     </FoundationalVersion>
+    <MicrosoftExtensionsHttpVersion>8.0.*</MicrosoftExtensionsHttpVersion>
```

Comment:
I did this because at least one of the packages that uses `FoundationalVersion` was resolving to 8.0.6 (or higher) and the package I needed only comes in 8.0.0. Looking at more examples (like Steeltoe.Common), we might need to add a few more variables to capture the variety that is currently resolving. I'm OK with using `FoundationalVersion` here if you've got an idea how to tackle that problem, otherwise it seems like using it now just adds another necessary change later

Reply:
After pulling your latest changes, I don't see usage of `MicrosoftExtensionsHttpVersion` anymore, but the variable still exists in this file.

Reply:
remove it now